docs/extensibility.md

```diff
@@ -597,77 +422,271 @@ kubectl expose deployment/device \
 >
 > This should return a list of 9 devices, of the form `http://device-X:8080`
 
-
-# Where the rubber meets the road!
-At this point, we've extended Akri to include discovery for our HTTP protocol and we've created an HTTP broker that can be deployed.  Let's take HTTP for a spin!!
-
 ## Deploy Akri
+Now that we have created a HTTP Discovery Handler and created some mock devices, let's deploy Akri and see how it
+discovers the devices and creates Akri Instances for each Device.
 
 > Optional: If you've previous installed Akri and wish to reset, you may:
 >
 > ```bash
 > # Delete Akri Helm
 > sudo helm delete akri
->
-> # Delete Akri CRDs
-> kubectl delete crd/configurations.akri.sh
-> kubectl delete crd/instances.akri.sh
 > ```
 
-Deploy the revised (!) Helm Chart to your cluster:
+Akri has provided helm templates for custom Discovery Handlers and their Configurations. These templates are provided as
+a starting point. They may need to modified to meet the needs of a Discovery Handler. When installing Akri, specify that
+you want to deploy a custom Discovery Handler as a Daemonset by setting `customDiscovery.discovery.enabled=true`.
+Specify the container for that DaemonSet as the HTTP discovery handler that you built
+[above](###build-the-discoveryhandler-container) by setting `customDiscovery.discovery.image.repository=$DH_IMAGE`. To
+automatically deploy a custom Configuration, set `customDiscovery.enabled=true`. We will customize this Configuration to
+contain the discovery endpoint needed by our HTTP Discovery Handler by setting it in the `discovery_details` string of
+the Configuration, like so: `customDiscovery.discoveryDetails=http://discovery:9999/discovery`. We also need to set the
+protocol name the Discovery Handler will register under (`customDiscovery.discoveryHandlerName`) and a name for the
+Discovery Handler and Configuration (`customDiscovery.name`). All these settings come together as the following Akri
+installation command:
+> Note: Be sure to consult the [user guide](./user-guide.md) to see whether your Kubernetes distribution needs any
+> additional configuration.
+```bash
+  helm repo add akri-helm-charts https://deislabs.github.io/akri/
+  helm install akri akri-helm-charts/akri \
+  --set imagePullSecrets[0].name="crPullSecret" \
+  --set customDiscovery.discovery.enabled=true  \
+  --set customDiscovery.discovery.image.repository=$DH_IMAGE \
```

Comment:
Dh_image has repo and label.  It needs to be split for use with helm.

Reply:
Same for each helm command in here :)

Reply:
Nice catch! fixed 34e97dcf0